test/unit_test/bloc/coins_bloc_test.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:bloc_test/bloc_test.dart';
+import 'package:crypto_mobile_application/src/blocs/coins_bloc/coins_bloc.dart';
+import 'package:crypto_mobile_application/src/blocs/coins_bloc/coins_event.dart';
+import 'package:crypto_mobile_application/src/blocs/coins_bloc/coins_state.dart';
+import 'package:crypto_mobile_application/src/services/coin_service.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+
+import '../../mock_data/coin_service_mock.dart';
+
+void main() {
+  String error = 'Null check operator used on a null value';
```

Comment:
This is not a proper way to test errors being thrown from the service.

Reply:
I fixed